src/gatsby-browser.js

```diff
@@ -0,0 +1,9 @@
+import Raven    from "raven-js";
+
+exports.onClientEntry = () => {
+    if (window && process.env.NODE_ENV === "production" && window.sentry_config) {
+        window.Raven = Raven;
+
+        window.Raven.config(window.sentry_config.dsn, window.sentry_config.additional_config).install();
```

Comment:
Good work 👍 

you can get the plugin options like this too:
```
exports.onClientEntry = (_, pluginParams) => {
  console.log(pluginParams.dsn
}
```

you can remove all gatsby-ssr file thanks to that.